### PR TITLE
adds myself to the icon/ass codeowners

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
i want asstainer now

## Why It's Good For The Game
i like icons/ass

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -135,6 +135,7 @@
 
 /_maps/ @EOBGames @Maurukas @san7890 @ShizCalev
 /icons/ @Imaginos16 @Krysonism @Twaticus
+/icons/ass @Ghilker
```
Comment (JohnFulpWillard) [inline .github/CODEOWNERS:138]: Multiple owners?

Comment (Fikou) [inline .github/CODEOWNERS]: my brother in christ read what the comment says

Comment (Ghilker) [reply .github/CODEOWNERS]: maybe someone else want to be asstainer with me

**Comment by tralezab** [comment]
take a look at this pronto @Mothblocks

**Comment by notfrying1pans** [comment]
what

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -96,6 +96,9 @@
 /code/modules/mining/ @Fikou
 /code/modules/awaymissions/ @Fikou
 
+# Ghilker
+/icons/ass/ @Ghilker @magatsuchi
```
Comment (Ghilker) [inline .github/CODEOWNERS:100]:
```suggestion
/icons/ass/ @Ghilker
```

**Comment by Jolly-66** [comment]
Why is san being requested for review

**Comment by fippefi** [comment]
understandably important, someone has to take ownership of these assets

**Comment by Ghilker** [comment]
> Why is san being requested for review

300px-You_Wouldn't_Get_It

**Comment by Mothblocks** [comment]
Thank you @tralezab I have made an informed update based on your inspection

**Pull request merged**